app/javascript/entrypoints/sdk.js

```diff
@@ -72,6 +72,13 @@ const runSDK = ({ baseUrl, websiteToken }) => {
     widgetStyle: getWidgetStyle(chatwootSettings.widgetStyle) || 'standard',
     resetTriggered: false,
     darkMode: getDarkMode(chatwootSettings.darkMode),
+    welcomeHeading: chatwootSettings.welcomeHeading || '',
```

Comment:
I have a few suggestions for improving the config options naming for better clarity.

```
window.chatwootSettings = {
  // Message customization
  welcomeTitle: 'Need help?',
  welcomeDescription: "We're here to support you.",
  availableMessage: "We're online and ready to chat!",
  unavailableMessage: "We're currently offline.",

  // Feature toggles
  enableFileUpload: true, // showFilePicker → enableFileUpload
  enableEmojiPicker: true, // showEmojiPicker → enableEmojiPicker
  enableEndConversation: true, // allowEndConversation → enableEndConversation
};
```
```

Reply:
Done